src/Frost/GitPlugin.hs

```diff
@@ -0,0 +1,17 @@
+module Frost.GitPlugin where
+
+import Frost.Effects.Git
+import Frost.Plugin
+import Text.Pandoc
+import Polysemy
+import PolysemyContrib
+import Data.Map.Strict
+
+gitPlugin :: Member GitEffect r => Plugin r
+gitPlugin = Plugin "git:devs" (\_ -> return []) atm
```

Comment:
You might consider `justMetaPlugin` smart constructor

Reply:
I think this should be a content plugin

Reply:
or we could have both :)